agent-zero/skills/blender-3d-studio/scripts/queue-render.sh

```
@@ -0,0 +1,684 @@
+echo -e "📁 Output directory: $OUTPUT_DIR"
+echo -e "🎞️  Frame range: $FRAME_START - $FRAME_END"
+echo -e "🎨 Quality: $QUALITY"
+echo -e "🔢 Samples: $SAMPLES"
+echo -e "📺 Resolution: ${RESOLUTION_X}x${RESOLUTION_Y}"
+echo -e "⚙️  Engine: $ENGINE"
+echo -e "🏎️  Priority: $PRIORITY"
+echo -e "🌐 Distributed: $DISTRIBUTE"
+echo ""
+
+# Add job to queue
+local job_id=$(add_job_to_queue)
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)